models/issue_label.go

```diff
@@ -79,11 +81,26 @@ func GetLabelTemplateFile(name string) ([][3]string, error) {
 	return list, nil
 }
 
-// CalOpenIssues calculates the open issues of label.
+// CalOpenIssues returns the open issues of label.
```

Comment:
why return -> the function doesnt return anything

Reply:
Because it is not actually doing a real live calculation, it is returning what is already stored in the database, where as the CalOpenOrg issues is actually doing a live calculation based on database and what repos that user has access to at the moment. So to make distinction between the two.

Reply:
I changed it to be more clear